mmdet/models/dense_heads/fcos_head.py

```diff
@@ -225,6 +242,25 @@ def loss(self,
              gt_labels,
              img_metas,
              gt_bboxes_ignore=None):
+        """Compute loss of the head.
+
+        Args:
+            cls_scores (list[Tensor]): Box scores for each scale level
+                Has shape (N, num_points * num_classes, H, W).
+            bbox_preds (list[Tensor]): Box energies / deltas for each scale
+                level with shape (N, num_points * 4, H, W).
+            centernesses (list[Tensor]): Centerss for each scale level with
+                shape (N, num_points * 1, H, W).
+            gt_bboxes (list[Tensor]): each item are the truth boxes for each
```

Comment:
gt_bboxes (list[Tensor]): Ground truth bboxes for each image with shape (num_gts, 4) in [tl_x, tl_y, br_x, br_y] format.